rootfs/usr/local/bin/eks-update-kubeconfig

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Usage:
+#   eks-update-kubeconfig <cluster-short-name>|off
```

Comment:
Thoughts on calling it `export-eks-kubeconfig`?

Reply:
Nevermind, I see what you gave it this name because the underlying command is `aws eks update-kubeconfig`